test/TestAsset/DocumentBasedTestCase.php

```diff
@@ -153,6 +153,26 @@ class DocumentBasedTestCase extends TestCase
             'BelongsToBibliography'   => 1,
             'EmbargoDate'             => '2010-01-04',
         ],
+        'testdocA'  => [
```

Comment:
Können wir "Weighted" mit in die Namen aufnehmen? Ich finde es auch etwas irritierend, dass die Testdaten in einer anderen Klasse stecken. Das ist natürlich schon im alten Code so. Gibt es einen besseren Ansatz für die neuen Tests, der mit wenig Aufwand verbunden ist?

Reply:
Ich kann `weightedTestdoc...` verwenden. Ich könnte die vom Test verwendeten Dokumente natürlich auch separat in einer Helper-Methode innerhalb derselben Klasse erstellen.